.github/workflows/build-fd-image.yaml

```diff
@@ -8,20 +8,30 @@ on:
       - 'v*'
     paths:
       - ".github/workflows/build-fd-image.yaml"
+      - ".github/workflows/scan-docker-image-action.yaml"
       - "cmd/fluent-watcher/fluentd/**"
       - "cmd/fluent-watcher/hooks/**"
       - "pkg/filenotify/**"
 
 env:
-  FD_IMG: 'kubesphere/fluentd:v1.15.3'
-  ARCH: '-arm64'
-  FD_IMG_BASE: 'kubesphere/fluentd:v1.15.3-arm64-base'
+
+  FD_IMG: "kubesphere/fluentd:${{ github.ref_name }}"                                 # kubesphere/fluentd:v2.2.2
```

Comment:
Taking branch name / git tag  as the image tag as I couldn't figure out what to keep.

Reply:
Let's switch to the metadata action for docker